frontend/packages/kubevirt-plugin/src/components/modals/menu-actions-modals/delete-vm-modal.tsx

```diff
@@ -117,22 +124,31 @@ export const DeleteVMModal = withHandlePromise((props: DeleteVMModalProps) => {
                 onChange={() => setDeleteVMImport(!deleteVMImport)}
                 checked={deleteVMImport}
               />
-              Delete {VirtualMachineImportModel.label} Resource
+              {t('kubevirt-plugin~Delete {{vmiModelLabel}} Resource', {
+                vmiModelLabel: VirtualMachineImportModel.label,
+              })}
             </label>
           </div>
         )}
         {vmHasSnapshots && (
           <>
-            <strong>Warning: </strong>All snapshots of this virtual machine will be deleted as well.
+            <Trans t={t} ns="kubevirt-plugin">
+              <strong>Warning: </strong>All snapshots of this virtual machine will be deleted as
+              well.
```

Comment:
is this prettier? the formatting looks a bit weird...

Reply:
haha. Definitely prettier. I would never do that by choice.